zoo/services/templates/services/fields/tag_input.html

```diff
@@ -0,0 +1,12 @@
+<div id="tag-input" class="tag-input-container">
+  <tag-input></tag-input>
+</div>
+
+<script>
+  const tagList = [{% for tag in tags %}'{{ tag }}', {% endfor %}]
```

Comment:
Oof, this will break with an XSS the moment we add some tag like `C++ '17` or `InfoSec's project`.

Reply:
Unless is explicitly marked as Safe [Django will escape the caracters](https://docs.djangoproject.com/en/2.1/topics/security/#cross-site-scripting-xss-protection). So no XSS should be happening. Unless the sanitization is not correct, but at that point we need to think how much we trust the tool.

Reply:
`C++ '17;alert('test');` gets outputted as `C++ &#39;17;alert(&#39;test&#39;);`